### PR TITLE
[feature/KEYCLOAK-15976]: KEYCLOAK-15976 Latvian language support

### PR DESCRIPTION
**Pull request by OskarsPakers**
Giving this another shot as #5676 was closed with no activity.

### CODE REVIEW

**Comment by stianst** [comment]
Hi, sorry for the really late reply here. Are you still interested in contributing this? If so could you ask for some volunteers to review it on the dev mailing list?

**Comment by OskarsPakers** [comment]
Hi! Would some comment with approval in GitHub work? I could find Latvians to review, but they aren`t in Keycloak dev mailing list.

**Comment by stianst** [comment]
Yeah, that will be fine. It might also be an idea to switch to utf8 encoding as that simplifies review/maintenance of the translation.

This can be done like this:
https://github.com/keycloak/keycloak/blob/64717f650b11cbc06bb498bbc4d6d107c9aed180/themes/src/main/resources-community/theme/base/admin/messages/messages_ja.properties#L1-L3

File: themes/src/main/resources-community/theme/base/account/messages/messages_lv.properties [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# encoding: utf-8
+doSave=Saglab\u0101t
```
Comment (stianst) [inline themes/src/main/resources-community/theme/base/account/messages/messages_lv.properties:2]: Looks like you haven't changed the encoding to UTF-8, and the values are still using ISO-8859-1, there are various tools that can help you with transforming it automatically to UTF-8.

**Comment by stianst** [comment]
Did you get anywhere with getting someone to review? Also, worth dropping a message on the dev mailing list to ask if there happens to be someone from the community that would be happy to review this.

**Comment by OskarsPakers** [comment]
> Did you get anywhere with getting someone to review? Also, worth dropping a message on the dev mailing list to ask if there happens to be someone from the community that would be happy to review this.

I`ve converted files to utf8 encoding and asked a few fellows to check the translations.

**Review by stianst — APPROVED** [review]

**Pull request merged**